defaults/main.yml

```diff
@@ -167,15 +167,15 @@ rhel7cis_rule_3_3_9: true
 rhel7cis_rule_3_4_1: true
 rhel7cis_rule_3_4_2: true
 rhel7cis_rule_3_5_1_1: true
-rhel7cis_rule_3_5_1_2: true
+rhel7cis_rule_3_5_1_2: false # Docker needs IP Tables https://docs.docker.com/network/iptables/
```

Comment:
Shouldn't we configure these when we import the role for the ECS playbook though?